src/frontend/src/pages/FinancePage/FinanceComponents/ReimbursementRequestInfo.tsx

```diff
@@ -38,32 +38,42 @@ const ReimbursementRequestInfo = ({
   allReimbursementRequests,
   canViewAllReimbursementRequests = false
 }: ReimbursementRequestInfoProps) => {
-  const [isAscendingOrder, setAscendingOrder] = useState(true);
-  const [orderBy, setOrderBy] = useState<keyof ReimbursementRequestRow>('dateSubmittedToSabo');
+  // set default values for orderBy and sortBy
+  if (!localStorage.getItem('orderBy')) localStorage.setItem('orderBy', 'identifier');
+  if (!localStorage.getItem('sortBy')) localStorage.setItem('sortBy', 'descending');
+
+  // set orderBy and sortBy from local storage
+  const [isAscendingOrder, setAscendingOrder] = useState(localStorage.getItem('ascendingOrder') === 'ascending');
```

Comment:
You're not using localStorage.ascendingOrder anymore, you're using sortBy (which is a confusing name, maybe sortDirection? might be better, see Sean's comment). 
Because of the name mismatch if I change it to ascending it won't load properly on refresh and it will override to descending (try it!). Please fix this bug.